ast/js.py

```diff
@@ -50,16 +50,16 @@ def _js_type(self, type, plural):
   def field(self, type, name, nullable, plural):
     nullable_char = '?' if nullable else ''
     js_type = self._js_type(type, plural)
-    print '  %(name)s%(nullable_char)s: %(nullable_char)s%(js_type)s;' % locals()
+    print('  %(name)s%(nullable_char)s: %(nullable_char)s%(js_type)s;' % locals())
 
   def start_union(self, name):
-    print ('type %s = ' % name),
+    print(('type %s = ' % name), end=' ')
```

Comment:
shouldn't this be `end=''`? (empty string)

Reply:
I choose to let 2to3.py do its magic and simply correct incompatibilities.
But yeah, this one is strange, I'll look it up.